### PR TITLE
Implement prototype mapping from extractors to LRMI fields

### PR DESCRIPTION
**Pull request by MRuecklCC**
First step on resolving the #94 issue. Implementation following [this comment](https://github.com/openeduhub/metalookup/issues/94#issuecomment-1191323792).

@torsten-simon Once merged and deployed, we can start testing the integration with the EduSharing service.

### CODE REVIEW

File: src/metalookup/app/api.py [hunk 1]
```diff
@@ -73,6 +73,51 @@ async def extract(input: Input, request: Request, response: Response, extra: boo
...
+
+    result = await manager.extract(input, extra=True)
+
+    def combine(*fields: str) -> Union[MetadataTags, Error]:
+        """
+        Combines the extractors' MetaDataTags for given extractors (the field names used in the Output model).
+        Star rating will be deduced as minimum star rating.
+        If any extractor failed, the combined result is also a failure listing all sub-failures.
+        """
+        fields: Dict[str, Union[Error, MetadataTags]] = {field: getattr(result, field) for field in fields}
```
Comment (RMeissnerCC) [inline src/metalookup/app/api.py:99]: If fields is a string, how can you iterate over it?

Comment (RMeissnerCC) [reply src/metalookup/app/api.py:99]: Ooh you unpack the given list. Mmmh, interesting way, but also quite explizit. I had to get to line 115 to understand line 93 and thus line 99.

**Review by RMeissnerCC — APPROVED** [review]
For a first draft nice. With more work and needs here I see some places for refactoring. But nice to get feedback from customer :)

File: src/metalookup/app/api.py (hunk 1 above)
Comment (MRuecklCC) [reply src/metalookup/app/api.py:99]: The function signature `def combine(*fields: str)` defines fields as `tuple[str,...]`. When using `*args` python always assumes that they are of the same type, and you only annotate the type that is in the tuple :)

**Pull request merged**